pkgs/by-name/ol/olympus/unwrapped.nix

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
This file should be in `pkgs/by-name/ol/olympus-unwrapped/package.nix` so that it is globally accessible outside of the wrapper. Then you can remove the `callPackage` invocation in the `olympus/package.nix` file

Reply:
Do you think it is better to have a `olympus-unwrapped` package or have it available in the passthru like `olympus.passthru.unwrapped`?